bot/exts/christmas/adventofcode.py

```diff
@@ -608,10 +845,13 @@ def from_json(cls, injson: dict) -> "AocPrivateLeaderboard":
         )
 
     @classmethod
-    async def from_url(cls) -> "AocPrivateLeaderboard":
+    async def from_url(cls, leaderboard_id: int, cookie: str) -> typing.Union["AocPrivateLeaderboard", None]:
         """Helper wrapping of AocPrivateLeaderboard.json_from_url and AocPrivateLeaderboard.from_json."""
-        api_json = await cls.json_from_url()
-        return cls.from_json(api_json)
+        api_json = await cls.json_from_url(leaderboard_id, cookie)
+        if api_json is not False:
+            return cls.from_json(api_json)
+        else:
+            return None
```

Comment:
Everywhere else returns `None` implicitly
```suggestion
            return
```